novice-r-intro.Rmd

```diff
@@ -2,27 +2,108 @@
 
 # Introduction {#novice-r-intro}
 
+**Note: we will include this introduction in the novice Python material as well.**
+
 FIXME: general introduction.
 
 ## Who are these lessons for? {#novice-r-intro-personas}
 
 FIXME: personas
 
-### Summary
+## What will these lessons teach you? {#novice-r-intro-goals}
+
+During this course, you will learn how to:
 
-FIXME: elevator pitch
+-   Write programs in R that read data, clean it up, perform simple statistical analyses on it, and plot the results.
+-   Find and install R packages to help you do these things.
+-   Create software that other people can understand and re-run.
+-   Track your work in version control using Git and GitHub.
+-   Publish your results in a web site using R Markdown and GitHub Pages.
+-   Make your data, software, and reports citable using ORCIDs and DOIs, and cite the work of others.
+-   Select open source and Creative Commons licenses that allow you and others to share data, software, and reports.
+-   Be an active participant in open, inclusive projects.
 
-### Prerequisites
+### What do you need to have and know to start? {#novice-r-intro-prereq}
 
-FIXME: prerequisites
+This course assumes that you:
 
-## What does "done" look like? {#novice-r-intro-goals}
+-   Have a laptop that you can install software on, or access to a web-based programming system like [rstudio.cloud][rstudio-cloud].
+-   Can calculate mean, variance, and correlation using a spreadsheet or similar tool.
+-   Are willing to invest about 30 hours in reading or lectures and another 100 hours doing practice exercises.
 
-FIXME: end goal
+We have tried to make these lessons accessible to people with visual or motor challenges,
+but recognize that some parts (particularly data visualization) may still be difficult.
+We welcome suggestions for improvements.
 
-## What will we use as running examples? {#novice-r-intro-example}
+## What examples will we use? {#novice-r-intro-example}
 
 FIXME: introduce running examples
 
+## What's the big picture? (#novice-r-intro-bigpicture}
```

Comment:
Should big picture also include why learning these skills are important? Aka, why they *aren't* just "merely useful".